packages/app-web/src/hooks/useShortcutHandler.ts

```diff
@@ -3,7 +3,7 @@ import { useEffect } from 'react'
 
 /**
  * Register a keydown event handler to the given element and hand over the
- * pressed key combinations using a shortcut notation.
+ * pressed key combinations using a shortcut notation. In other words, listens to keyboard events, compiles the shortcut notation and calls onShortcut
  */
```

Comment:
If you don't like my phrasing here, rephrase and replace it instead of adding an alternative at the end. Less is more 😉

Reply:
Hahaha, I wasn't quite sure about that. I'll go back to the drawing board on that. Might also be worth adding the phrase "Less is more" to CONTRIBUTING.md

Reply:
I'm not sure, people may take it the wrong way without context 🙈